src/rhelocator/update_images/aws.py

```diff
@@ -114,3 +114,65 @@ def parse_image_name(image_name: str) -> dict[str, str]:
         return matches.groupdict()
 
     return {}
+
+
+def format_all_images() -> object:
```

Comment:
I don't like returning 'object' but the literal type annotation would be 'dict[str, dict[str, list[dict[str, str]]]]' ..
Anybody got a better idea?

Reply:
Haha, no better ideas here. This is where typing gets a little challenging.